builder/service.go

```diff
@@ -144,9 +155,27 @@ func Register(stack *node.Node, backend *eth.Ethereum, cfg *Config) error {
 		localRelay = NewLocalRelay(relaySk, beaconClient, builderSigningDomain, proposerSigningDomain, ForkData{cfg.GenesisForkVersion, cfg.BellatrixForkVersion, cfg.GenesisValidatorsRoot}, cfg.EnableValidatorChecks)
 	}
 
+	relayConfigs := make(map[string]RelayConfig)
+	if cfg.RelayConfigFile != "" {
+		bytes, err := os.ReadFile(cfg.RelayConfigFile)
+		if err != nil {
+			log.Info("Failed to read relay config file, will use default settings", "err", err)
```

Comment:
should the builder not fail here? i.e. if a config file was explicitly requested, it should never fall back to other configuration, as it's not what the user specifically requested 🤔